Week1/Substringtest.java

```diff
@@ -0,0 +1,60 @@
+import static org.testng.Assert.assertEquals;
+
+import org.testng.annotations.Test;
+
+public class Substringtest {
+  LongestSubString s=new LongestSubString();
```

Comment:
Your subString method is static, meaning you do not need to instantiate LongestSubstring here